README.md

```diff
@@ -43,21 +43,29 @@ module.exports = (inputStreams, outputStreams) => {
     // do something
 };
 module.exports.$interactionModel = 'node-streams';
-module.exports.$arity = 3;
 ```
-The interaction mode and the arity are **required** in this case.
 
-The arity is the number of input streams plus the number of output streams the function accepts
-(here: 1 input stream + 2 output streams hence an arity of 3).
+Please note that streaming functions must always declared the corresponding interaction mode.
+
+Parameters can also be looked up by name:
+```js
+module.exports = (inputStreams, outputStreams) => {
+    const { numbers, letters } = inputStreams;
+    const letters = inputStreams["letters"];
```

Comment:
rm

Reply:
Fun fact: Jetbrains' WebStorm actually outlines that line in red.